pkg/users/accounts-list.js

```diff
@@ -93,7 +93,7 @@ const UserActions = ({ account }) => {
     return kebab;
 };
 
-const getGroupRow = (group, accounts) => {
+const getGroupRow = (group, accounts, isUserCreatedGroup) => {
```

Comment:
This "show renaming and deleting actions only for the user created groups" commit should be squashed. It also needs an integration test that confirms that the rename action is not present in the menu. Also see above wrt. showing an empty kebab.